config/purposes/bioscan.yml

```diff
@@ -44,23 +39,61 @@ LBSN-96 Lysate:
     - type: well_exclusions
       value:
         - H12
+  :presenter_class: Presenters::SubmissionWhenPassedPlatePresenter
+  :submission_options:
+    Bioscan Library Prep:
+      template_name: 'Limber-Htp - Bioscan Library Prep - Automated'
+      request_options:
+        library_type: 'Bioscan'
+        read_length: 150
+        fragment_size_required_from: '50'
+        fragment_size_required_to: '500'
+      allowed_extra_barcodes: false
+LBSN-96 Lysate Input:
+  :asset_type: plate
+  :input_plate: true
+  :stock_plate: true
+  :cherrypickable_target: false
+  :label_template: plate_96_lysate
+  :creator_class: LabwareCreators::Uncreatable
+  :size: 96
+  :presenter_class: Presenters::SubmissionPlatePresenter
+  :submission_options:
+    Bioscan Library Prep:
+      template_name: 'Limber-Htp - Bioscan Library Prep - Automated'
+      request_options:
+        library_type: 'Bioscan'
+        read_length: 150
+        fragment_size_required_from: '50'
+        fragment_size_required_to: '500'
+      allowed_extra_barcodes: false
 LBSN-384 PCR 1:
   :asset_type: plate
   :creator_class: LabwareCreators::QuadrantStamp
   :merger_plate: true
   :presenter_class: Presenters::MinimalPcrPlatePresenter
   :label_template: plate_384_single
   :size: 384
-  # LYSATE plate barcode is shown on barcode label top-right
-  :alternative_workline_identifier: LBSN-96 Lysate
+  # LYSATE plate barcode is shown on barcode label top-right, user does not want
+  # to see the LILYS plate barcode here.
+  # This field indicates the purpose of the ancestor we want to find to show on
+  # the barcode label. We have 2 possible ancestors.
+  :alternative_workline_identifier:
+    - LBSN-96 Lysate
+    - LBSN-96 Lysate Input
```

Comment:
Note: This array syntax is a useful addition. It provides a better controlled way of adding the ancestor barcode to the label.